monkeys/fqdn-s3/run.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+ERROR_MSG="TODO: Update the error message"
+
+. $(dirname ${BASH_SOURCE})/helpers.bash
+
+init_monkey
+
+[ -z "$SLEEP" ] && {
+    SLEEP="10"
+}
+
+[ -z "$CONNECTTIMEOUT" ] && {
+    $CONNECTTIMEOUT="5"
+}
+endpoint_debug_enable
+
+while :
+do
+    start_monitor
+    start_tcpdump
+    curl --head --fail --connect-timeout "${CONNECTTIMEOUT}" "${TARGETURL}" \
+        -w "RemoteIP: %{remote_ip}\nDNSLOOKUP: %{time_namelookup}"
```

Comment:
It might be good to include the total time as well, as a way to check that the network wasn't being slow overall. The HTTP return code might also be useful, in case we hit a rate limit and see a 429 (but this might not happen).